lerobot/scripts/control_robot.py

```diff
@@ -798,7 +799,13 @@ def replay(robot: Robot, episode: int, fps: int | None = None, root="data", repo
         nargs="*",
         help="Any key=value arguments to override config values (use dots for.nested=overrides)",
     )
-
+    parser_record.add_argument(
```

Comment:
Does this matter? It gets lost in the hub upload anyway right? Then when someone wants to actually use the dataset they can set the tolerance as they wish.

Reply:
It's weird that this is the case as you are not using `delta_timestamps` which is the only way I could see the tolerance guard is being triggered. Do you have an exception trace (only dig it up if you cbb, as this PR is on hold anyway).

Reply:
no I don't unfortunately :(